### ISSUE TITLE
Asus Z170I PRO Gaming - Fan Speeds

### ISSUE THREAD

**Issue by iamwyza** [openhardwaremonitor/openhardwaremonitor#815]
Would it be possible for OHM to report the fan speeds off of the Asus Z170I Pro Gaming?  I've downloaded 0.7.1.4 Alpha, which thankfully now reports tempuratures, but not fan speeds.

Thanks for your efforts!

**Comment by DarthDestroyer** [comment]
Could you try the latest Version?

http://openhardwaremonitor.org/files/openhardwaremonitor-v0.7.1.5-alpha.zip

**Comment by iamwyza** [comment]
No dice.
![image](https://cloud.githubusercontent.com/assets/1375890/16173522/e8aca698-3568-11e6-9775-56534fc00544.png)

**Comment by DarthDestroyer** [comment]
First: No Screenshot ;-) A Report... But that is my Fault! Click on File - > Report and you will get a textfile ;-)

Second: I think, you are not alone! When i am correct,  then there is already a z170 issue...

**Comment by iamwyza** [comment]
Thanks. See attached.
[OpenHardwareMonitor.Report.txt](https://github.com/openhardwaremonitor/openhardwaremonitor/files/322428/OpenHardwareMonitor.Report.txt)

**Comment by erikwalfridsson** [comment]
> Second: I think, you are not alone! When i am correct, then there is already a z170 issue... 
Yes, there are several Z170 issues, but isn´t there a seperate chip for the hardware monitoring? I didn´t think that part was integrated into chipset itself. 

It would be good if someone more knowledgeable could clarify this.

I have the same problem with Asus Z170 pro gaming (not Z170**I** pro gaming), and 0.7.1.4 Alpha but I can´t try with 0.7.1.5 Alpha and post my report now , because that computer is taken apart.

**Comment by DarthDestroyer** [comment]
For that, we could need the Report ;-)

The Monitoring "chip" is often the same, in a Series of Mainboards. It could be another, but that information is accessible through the report! 

Feel free to look at iamwyza's report, to know what i mean... 

So, for this... We could need your report ;-)

Kind Regards

**Comment by ewwink** [comment]
0xD121 = NCT6793D

**Comment by Callidus80** [comment]
I have the same issue. MSI Z170A Gaming M5 with Same Chip ID 0xD121. Hope you have time to include in the next release. 
Also : KEEP UP THE CRACKING WORK ! Love Open Hardware Monitor :)
[OpenHardwareMonitor.Report.txt](https://github.com/openhardwaremonitor/openhardwaremonitor/files/887830/OpenHardwareMonitor.Report.txt)
